docs/home/whats-new-2-13.md

```diff
@@ -23,17 +24,21 @@ This version contains features and fixes from previous versions starting with 2.
 
 <!-- RUN-9359/9360 Incorporating Node Pools in Workspaces -->
 **Nodes**
+<!-- Hagay: not sure I understand why did we had this intro paragraph, and anyway the changes are in node-pools not in nodes (other than the DCGM metrics of 2.10/2.11 but we don't mention it here). 
+Also the headeline is 'Nodes' but the content is 'Node-Pools' -->
 
 * A node is a worker machine that runs workloads, and a node pool is group of nodes within a cluster that all have the same configuration. Node pools use node labels as its identification. Run:ai has added a table for viewing nodes and for configuring node pools. To configure a node pool, see [Configuring node pools](../Researcher/scheduling/using-node-pools.md#creating-new-node-pools).
 
 <!-- RUN-9960/9961 Per node-pool GPU placement strategy -->
 * Run:ai scheduler supports 2 scheduling strategies: Bin Packing (default) and Spread. For more information, see [Scheduling strategies](../Researcher/scheduling/strategies.md). You can configure the scheduling strategy in the node pool level to improve the support of clusters with mixed types of resources and workloads. For configuration information, see [Creating new node pools](../Researcher/scheduling/using-node-pools.md#creating-new-node-pools).
+<!-- Hagay: MISSING feature - Per node-pool Over-Quota Priority 
+ Run:ai Project allows and Admin to configure if the Project should support Over-Quota/Over-Quota-Priority, this capability is now enhanced to allow the Administrator to configure Over-Quota/Over-Quota-Priority per node-pool per project, to read more about Over-Quota/Over-Quota-Priority go <here> -->
 
+<!-- Hagay: Jason, the node pools priority is new to Workspaces but was already existing on Workloads(Interactive Jobs, Training) -->
 <!-- RUN-10287/10317/10313-10851 Show Node pools priority list according to workspace policy -->
```

Comment:
@jasonnovichRunAI Add section called Asset based workloads. Move this to there. workspace submission form - remove the workspace and make it submission form.